engine/src/main/scala/cromwell/engine/db/package.scala

```diff
@@ -18,6 +59,41 @@ package object db {
           (e.fqn == fqn) && e.index.isDefined
         })
     }
+
+    def toCallOutputs(callLogs: CallLogs): CallOutputs = {
+      import ExecutionDatabaseKey._
+      val callLogMap = Map(StdoutSuffix -> callLogs.stdout, StderrSuffix -> callLogs.stderr) ++
+        callLogs.backendLogs.getOrElse(Map.empty)
+
+      callLogMap map {
+        case (suffix, value) => s"${CallLogPrefix}_${attempt}_$suffix" -> CallOutput(value, None)
+      }
+    }
+
+    def toCallLogs(entries: Traversable[SymbolStoreEntry]): Option[CallLogs] = {
+      import ExecutionDatabaseKey._
+
+      var maybeStdout: Option[WdlFile] = None
+      var maybeStderr: Option[WdlFile] = None
+      var logs = Map.empty[String, WdlFile]
```

Comment:
seems like a fold with a case class accumulator having these three fields could do it

```scala
case class Acc(stdout: Option[WdlFile] = None, stderr: Option[WdlFile] = None, logs: Map[String, WdlFile] = Map.empty)
```